#! /bin/sh /usr/share/dpatch/dpatch-run
## sys_readline.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use the system copy of editline

@DPATCH@
diff -urNad asterisk-1.2.7.1.dfsg/Makefile /tmp/dpep.nfcwXR/asterisk-1.2.7.1.dfsg/Makefile
--- asterisk-1.2.7.1.dfsg/Makefile	2006-04-12 00:58:47.000000000 +0300
+++ /tmp/dpep.nfcwXR/asterisk-1.2.7.1.dfsg/Makefile	2006-05-13 22:54:43.051344312 +0300
@@ -305,7 +305,8 @@
 
 endif # WITHOUT_ZAPTEL
 
-LIBEDIT=editline/libedit.a
+#LIBEDIT=editline/libedit.a
+LIBS+=-ledit
 
 ifneq ($(wildcard .version),)
   ASTERISKVERSION:=$(shell cat .version)
@@ -513,7 +514,7 @@
 cygwin_a:
 	$(MAKE) -C cygwin all
 
-asterisk: $(CYGLOADER) editline/libedit.a db1-ast/libdb1.a stdtime/libtime.a $(OBJS)
+asterisk: $(CYGLOADER) db1-ast/libdb1.a stdtime/libtime.a $(OBJS)
 	build_tools/make_build_h > include/asterisk/build.h.tmp
 	if cmp -s include/asterisk/build.h.tmp include/asterisk/build.h ; then echo ; else \
 		mv include/asterisk/build.h.tmp include/asterisk/build.h ; \
